src/main/java/com/btkelly/gnag/api/GitHubApi.java

```diff
@@ -70,12 +75,24 @@ public Response intercept(Chain chain) throws IOException {
             }
         });
 
+        if (gnagPluginExtension.debugLogEnabled()) {
+            HttpLoggingInterceptor interceptor = new HttpLoggingInterceptor();
+            interceptor.setLevel(HttpLoggingInterceptor.Level.BODY);
```

Comment:
It doesn't, maybe it should? I'd have to extend the HttpLoggingInterceptor but I don't think it would be difficult.